index.js

```diff
@@ -195,6 +202,12 @@ function isSourceElement(node) {
     case "TypeAliasDeclaration": // Typescript
     case "ExportAssignment": // Typescript
     case "ExportDeclaration": // Typescript
+    case "ObjectExpression": // JSON
```

Comment:
Can you only do that when you're in a json context? Otherwise it's going to format weird things inside of js.